low-level/sspl_ll_d

```diff
@@ -777,6 +788,9 @@ if __name__ == "__main__":
     signal.signal(signal.SIGUSR2, refresh_iem_log_file)
     #DO NOT EDIT: Marker comment to dynamically add signal handler for SIGUSR1 to generate code coverage report
 
+    # Initialize IEM framework
```

Comment:
Needed under try-exception block. And should SSPL exit if this fails, probably not, to continue allowing other imp monitoring. Log it as critical, in bold about IEM and continue

Reply:
Ack.
logging the error with critical label and sspl will continue